### ISSUE TITLE
Редактирование вики мапии через JOSM

### ISSUE THREAD

**Issue by Sowa1980** [korint/wikimapia.evolution#12]
Редактирование вики мапии через JOSM, в разы повысит качество и скорость внесение данных и удобство по внесению данных, также позволит подключать собственные подложки и снимки, также позволит использовать встроенные подложки спутниковых снимков и карт. JOSM - редактор с открытым исходным кодом позволяющий это сделать.

**Comment by korint** [comment]
Есть ли выгрузка нарисованного в файл из проги?

**Comment by Anakin86** [comment]
> Есть ли выгрузка нарисованного в файл из проги?

Да, в JOSM можно сохранять нарисованное в локальный файл в формате OSM XML. То есть на самом деле нужно сделать поддержку импорта/экспорта в этот формат данных из Викимапии по заданной области.

**Comment by Sowa1980** [comment]
> Есть ли выгрузка нарисованного в файл из проги?

Если Вам нужен файл, могу прислать или можете сохранить любую область сами. через редактор. Форматов в какие сохранить есть разнообразные. выберите какой нужен Вам.

**Comment by korint** [comment]
Да, нужен пример объекта с названием на языках и куска дороги.
